### PR TITLE
multiProject1 Xcode workspace + systemTests scripts refactor

### PR DESCRIPTION
**Pull request by brunobowden**
multiProject1 Xcode workspace and project …
- Default configuration with no adjustments
- iOS Application: “Single View Application”
- No watchOS or OS X targets
- .gitignore extended to cover Xcode (copied from github/ignore repo)

Simpler developer running of tests …
- Run all scripts from root directory, without needing push systemTests
- Separate out runTest method as run-test.sh script
- Push J2OBJC_VERSION in to script, can be set by user but still has default

### CODE REVIEW

**Comment by brunobowden** [comment]
I will follow this up with more changes.

File: systemTests/multiProject1/ios/multiProject1/multiProject1.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -0,0 +1,292 @@
...
+			name = LaunchScreen.storyboard;
+			sourceTree = "<group>";
+		};
+/* End PBXVariantGroup section */
+
+/* Begin XCBuildConfiguration section */
+		931087921BC4B349005A0345 /* Debug */ = {
+			isa = XCBuildConfiguration;
+			buildSettings = {
+				ALWAYS_SEARCH_USER_PATHS = NO;
```
Comment (advayDev1) [inline systemTests/multiProject1/ios/multiProject1/multiProject1.xcodeproj/project.pbxproj]: should these be in xcconfigs instead?

File: systemTests/multiProject1/ios/multiProject1/multiProject1/AppDelegate.swift [hunk 2]
```diff
@@ -0,0 +1,46 @@
+//
+//  AppDelegate.swift
+//  multiProject1
+//
+//  Created by Bruno Bowden on 10/6/15.
```
Comment (advayDev1) [inline systemTests/multiProject1/ios/multiProject1/multiProject1/AppDelegate.swift]: can you replace this and the line below with our standard apache stanza?

File: systemTests/multiProject1/ios/multiProject1/multiProject1/ViewController.swift [hunk 3]
```diff
@@ -0,0 +1,25 @@
+//
+//  ViewController.swift
+//  multiProject1
+//
+//  Created by Bruno Bowden on 10/6/15.
```
Comment (advayDev1) [inline systemTests/multiProject1/ios/multiProject1/multiProject1/ViewController.swift]: ditto re: apache

**Comment by advayDev1** [comment]
i like it.  just a few comments and lgtm.

File: systemTests/multiProject1/ios/multiProject1/multiProject1.xcodeproj/project.pbxproj (hunk 1 above)
Comment (brunobowden) [inline systemTests/multiProject1/ios/multiProject1/multiProject1.xcodeproj/project.pbxproj]: This is the default configuration when the workspace and target was created. The goal is default and unmodified setup here, then all the changes are in a future PR.

**Comment by brunobowden** [comment]
Fixed copyright notice with standard Apache stanza.

**Pull request merged**